versionedController/v1/login/login.go

```diff
@@ -48,22 +48,19 @@ func (login *LoginController) Post(c *gin.Context) {
 		})
 		return
 	}
-
 	controller.Server.LocalLoginRequest(c, loginModel.Username, loginModel.Password)
-	return
 }
 
-/* Get will be triggered on GET request on the same path as Login alogn with a "auth_type" parameter
-** so as to identify the type of login user is up to. This has to be triggered through a href request
-** so that the user is able to be redirected to provider page for login.
-** Javascript Get Request can block the redirection of user */
+// Get will be triggered on GET request on the same path as Login alogn with a "auth_type" parameter
+// so as to identify the type of login user is up to. This has to be triggered through a href request
+// so that the user is able to be redirected to provider page for login.
+// Javascript Get Request can block the redirection of user
```

Comment:
Why this change?
Multi-line comment (ealier one) seems better?

Reply:
golang recommends the function/method level comments to be like this. The standard library and most other code document their code this way. It's kind of weird but this is the most common way to write function/method level docs.

Reply:
Ok